specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/bigDataPool.json

```diff
@@ -498,6 +498,10 @@
           "format": "date-time",
           "description": "The time when the Big Data pool was updated successfully.",
           "readOnly": true
+        },
```

Comment:
do this change not required in stable version of API.

Reply:
updated in stable version